### DISCUSSION TITLE
get parent records id for relation before creation

### DISCUSSION THREAD

**Discussion by rehmaninvozone** [prisma/prisma#16954] [Q&A]
Hi, is there any way to get the user id and store it in relation like I want to get the user id in companies relation for ownerId? or do I have to use another query to update the record?
```
   const user = await this.prisma.user.create({
        data: {
          ...payload,
        
          companies: {
            create: [
              {
                ownerId:  // here I want the parent id
                name: payload.firstName.trim().toLowerCase(),
              },
            ],
          },
        },
      });

**Comment by nurul3101** [comment]
Hello @rehmaninvozone 👋 

Can you please elaborate on what you are trying to achieve? I am not able to fully understand what you are trying to achieve.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
